supabase/functions/upload-to-storage/index.ts

```diff
@@ -0,0 +1,198 @@
+        textColor: 0,
+      },
+      head: [["Summary of Qualification"]],
+      body: [[employee.summary || ""]],
+    });
+
+    autoTable(doc, {
+      theme: "grid",
+      tableLineColor: [192, 192, 192],
+      tableLineWidth: 1,
+      styles: {
+        lineColor: [192, 192, 192],
+        lineWidth: 1,
+      },
+      headStyles: {
+        fillColor: [224, 224, 224],
+        fontSize: 15,
+        halign: "center",
+        textColor: 0,
+      },
+      bodyStyles: {
+        fillColor: [250, 250, 250],
+        textColor: 0,
+      },
+      head: [
+        [
+          {
+            content: "Technical Skills",
+            colSpan: 2,
+          },
+        ],
+      ],
+      body: [
+        ["Programming languages", employee.programming_languages],
+        ["Libs & Frameworks", employee.libs_and_frameworks],
+        ["Big Data", employee.big_data],
+        ["Databases", employee.databases],
+        ["Dev Ops", employee.devops],
+      ],
+    });
+
+    autoTable(doc, {
+      theme: "grid",
+      tableLineColor: [192, 192, 192],
+      tableLineWidth: 1,
+      styles: {
+        lineColor: [192, 192, 192],
+        lineWidth: 1,
+      },
+      headStyles: {
+        fillColor: [224, 224, 224],
+        fontSize: 15,
+        halign: "center",
+        textColor: 0,
+      },
+      bodyStyles: {
+        fillColor: [250, 250, 250],
+        textColor: 0,
+      },
+      head: [
+        [
+          {
+            content: "Education",
+            colSpan: 2,
+          },
+        ],
+      ],
+      body: [
+        [
+          "University degree",
+          `${employee.university}\n${employee.degree}\n${employee.university_start} - ${employee.university_end}`,
+        ],
+        [
+          "Level of English \nSpoken \nWritten",
+          `\n${employee.english_spoken}\n${employee.english_written}`,
+        ],
+        ["Certifications", employee.certifications],
+      ],
+    });
+
+    autoTable(doc, {
+      theme: "grid",
+      tableLineColor: [192, 192, 192],
+      tableLineWidth: 1,
+      styles: {
+        lineColor: [192, 192, 192],
+        lineWidth: 1,
+      },
+      headStyles: {
+        fillColor: [224, 224, 224],
+        fontSize: 15,
+        halign: "center",
+        textColor: 0,
+      },
+      bodyStyles: {
+        fillColor: [250, 250, 250],
+        textColor: 0,
+      },
+      head: [
+        [
+          {
+            content: "Additional Information",
+            colSpan: 2,
+          },
+        ],
+      ],
+      body: [["Personal qualities", employee.personal_qualities]],
+    });
+
+    const result = doc.output("arraybuffer");
+    // const result = 'test!!';
+
+    const uploadName = `${employee.first_name}-${employee.last_name
+      }-CV`;
+    const { data: upload } = await supabaseClient.storage
+      .from("pdfs")
+      .upload(uploadName, result, {
+        contentType: "application/pdf",
+        // contentType: 'text/html',
```

Comment:
If this is not planned on being used, please delete it. If there is a case in which we could use this contentType, we should add comments on when/why.